tfe/data_source_policy_set.go

```diff
@@ -33,6 +33,17 @@ func dataSourceTFEPolicySet() *schema.Resource {
 				Computed: true,
 			},
 
+			"kind": {
```

Comment:
Would be nice to add descriptions here.  I know it won't be consistent with the rest of the file